src/reporters/CloudNotifierReporter.ts

```diff
@@ -58,34 +59,34 @@ export class LambdaReporter extends Reporter {
   }
 
   reportCheck(ruleName: string, checkName: string, checkOptions: CheckOptions, outcome: Result, message?: string) {
-    let check_record: ResultRecord = {
+    let checkRecord: ResultRecord = {
```

Comment:
<details>
<summary>:warning: <strong>This code uses `let` or `var` to declare one or more variables that are never reassigned.</strong> (click for details)</summary>

A possible improvement would be to declare such a variable with `const` instead.   
  
This change would ensure that it would not be reassigned unintentionally, which could lead to difficulty to comprehend bugs.

**[More details](https://confluence.devfactory.com/x/UA_kEw)**

## The improvement suggestion in this case
```suggestion  
    const checkRecord: ResultRecord = {
```  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence.
</details>

[//]: # (<a id="9c51e063-027d-4ee4-aab1-7a2024d80939" data-finding-id="329518928" href="none" codereview-uuid-reserved-tag data-business-id="932b6095-04c8-4079-a6b1-1d6d1eb22183" data-analyzer="js" />)